aspnet/security/app-secrets.rst

```diff
@@ -14,74 +14,89 @@ This tutorial shows how your application can securely store and access secrets i
 
-For example, if you create a new ASP.NET web site app with individual user accounts, it will add a default connection string to the *config.json* file in the project with the key ``Data:DefaultConnection:ConnectionString``. The default connection string is setup to use LocalDB, which runs in user mode and doesn't require a password. When you deploy your application to a test or production server you can override the ``Data:DefaultConnection:ConnectionString`` key value with an environment variable setting that contains the connection string (potentially with sensitive credentials) for a test or production SQL Server. 
+For example, if you create a new ASP.NET Core web app with individual user accounts, it will add a default connection string to the *appsettings.json* file in the project with the key ``DefaultConnection``. The default connection string is setup to use LocalDB, which runs in user mode and doesn't require a password. When you deploy your application to a test or production server you can override the ``DefaultConnection`` key value with an environment variable setting that contains the connection string (potentially with sensitive credentials) for a test or production database server.
 
-.. note:: Environment variables are generally stored in plain text and are not encrypted. If the machine or process is compromised then environment variables can be accessed by untrusted parties. Additional measures to prevent disclosure of user secrets may still be required.
+.. warning:: Environment variables are generally stored in plain text and are not encrypted. If the machine or process is compromised then environment variables can be accessed by untrusted parties. Additional measures to prevent disclosure of user secrets may still be required.
 
 Secret Manager
 ^^^^^^^^^^^^^^
 
 The Secret Manager tool provides a more general mechanism to store sensitive data for development work outside of your project tree. The Secret Manager tool is a project tool that can be used to store secrets for a `.NET Core`_ project during development. With the Secret Manager tool you can associate app secrets with a specific project and share them across multiple projects.
 
-.. note:: The Secret Manager tool does not encrypt the stored secrets and should not be treated as a trusted store. It is for development purposes only.
+.. warning:: The Secret Manager tool does not encrypt the stored secrets and should not be treated as a trusted store. It is for development purposes only. The keys and values are stored in a JSON configuration file in the user profile directory.
 
 Installing the Secret Manager tool
```

Comment:
@blowdart  Given that Non-encryption is the #1 customer complaint, I want a warning  that visually stands out
![image](https://cloud.githubusercontent.com/assets/3605364/15440279/6084741c-1e70-11e6-857b-b8b63044f4a5.png)

>I'm fine with this.
I still need a [:]shipit[:}